server/src/data/models/base/base.model.js

```diff
@@ -0,0 +1,30 @@
+import { Model, snakeCaseMappers } from 'objection';
+
+class BaseModel extends Model {
```

Comment:
```suggestion
class Abstract extends Model {
```
don't forget about file name